Templates/QueueTrigger-FSharp/readme.md

```diff
@@ -0,0 +1,11 @@
+# QueueTrigger - F<span>#</span>
+
+The `QueueTrigger` makes it incredibly easy to react to new Queues inside of Azure Queue Storage. This sample demonstrates a simple use case of processing data from a given Queue using F#.
+
+## How it works
+
+For a `QueueTrigger` to work, you provide a path which dictates where the queue messages are located inside your container.
```

Comment:
I think the C# source you copied this from is also incorrect - it also talks about "path" and "container" which was likely copied from blob trigger. You don't specify a "path", you specify a queue name.